### PR TITLE
Update tests with the latest flow

### PR DESCRIPTION
**Pull request by tatiNo5**

### CODE REVIEW

**Review by VMitov — APPROVED** [review]

File: integration-tests/auth_utils.go [hunk 1]
```diff
@@ -33,35 +27,26 @@ func authenticate(httpClient *http.Client, identity identity, userID string, pin
...
-	if authResponse.Status != 200 {
-		return nil, fmt.Errorf("error making the authenticate request, status: %v", authResponse.Status)
+		return fmt.Errorf("error making the authenticate request: %w", err)
 	}
 
-	// Call to /rps/v2/access endpoint.
-	accessResponse, err = accessRequest(httpClient, authorizeResponse.WebOTT)
-	if err != nil {
-		return nil, fmt.Errorf("error making the access request: %w", err)
+	if authResponse.Status != 200 {
```
Comment (ZSPASOV) [inline integration-tests/auth_utils.go:45]: Shouldn't we change 200 with the http.StatusOK constant ?

File: integration-tests/auth_utils.go [hunk 2]
```diff
@@ -5,26 +5,20 @@ import (
...
 	rand := bindings.NewRand([]byte{})
 	X := make([]byte, 32)
 	proof := make([]byte, 65)
 	xR, S, U, _, err := wrap.Client1BN254CX(int(gomiracl.SHA256), 0, identity.MPinID, rand, X, pin, identity.Token, proof)
 	if err != nil {
-		return nil, fmt.Errorf("error getting client 1: %w", err)
+		return fmt.Errorf("error getting client 1: %w", err)
 	}
 
 	// Call to /rps/v2/pass1 endpoint.
```
Comment (ZSPASOV) [inline integration-tests/auth_utils.go:24]: The error on line 25 is not handled. I know it is not from the current pull request, it is an issue from before.

File: integration-tests/auth_utils.go (hunk 1 above)
Comment (tatiNo5) [reply integration-tests/auth_utils.go:45]: done

File: integration-tests/auth_utils.go (hunk 2 above)
Comment (tatiNo5) [reply integration-tests/auth_utils.go:24]: good catch. updated!

**Review by VMitov — APPROVED** [review]

**Review by ZSPASOV — APPROVED** [review]

**Review by VMitov — APPROVED** [review]

**Pull request merged**